src/com/facebook/buck/android/MergeAndroidResourcesStep.java

```diff
@@ -339,7 +339,7 @@ void writePerPackageRDotJava(
     SortedSetMultimap<RDotTxtEntry, Path> bannedDuplicateResourceToSymbolsFiles =
         TreeMultimap.create();
 
-    HashMap<RDotTxtEntry, String> resourceToIdValuesMap = new HashMap<>();
+    HashMap<Integer, RDotTxtEntry> resourceToIdValuesMap = new HashMap<>();
```

Comment:
we also pass this map to `getStyleableResources()` method where we retrieve actual RDotTxtEntry instance (which should contain proper "parent" information). Otherwise parent info will be lost here.

Changing to Map<RDotTxtEntry, RDotTxtEntry> makes sense - Ill make proper changes